Day-II-Calculator/calculator/src/App.js

```diff
@@ -9,30 +9,30 @@ const App = () => {
     <div className='container'>
       <div><CalculatorDisplay /></div>
       <div>
-        <ActionButton text='Clear' buttonStyle='threeQtr' />
-        <NumberButton text='/' buttonStyle='symbol'/>
+        <ActionButton text='clear' buttonStyle='threeQtr' />
+        <NumberButton buttonStyle="fas fa-divide" />
       </div>
       <div>
         <NumberButton text='7' />
         <NumberButton text='8' />
         <NumberButton text='9' />
-        <NumberButton text='X' buttonStyle='symbol'/>
+        <button className="fas fa-times"></button>
       </div>
       <div>
         <NumberButton text='4' />
         <NumberButton text='5' />
         <NumberButton text='6' />
-        <NumberButton text='-' buttonStyle='symbol'/>
+        <button className="fas fa-minus"></button>
       </div>
       <div>
         <NumberButton text='1' />
         <NumberButton text='2' />
         <NumberButton text='3' />
-        <NumberButton text='+' buttonStyle='symbol'/>
+        <button className="fas fa-plus"></button>
       </div>
       <div>
         <ActionButton text='0' buttonStyle='threeQtr' />
-        <NumberButton text='=' buttonStyle='symbol'/>
+        <button className="fas fa-equals"></button>
```

Comment:
Nice! Someone else used HTML operator symbols. It's good that you used the font awesome for that too :) Great work on getting the MVP done today. 👍 💯 💯